src/common/fullstory.ts

```diff
@@ -9,27 +7,36 @@ import * as FullStory from '@fullstory/browser';
  * and set the flag to true if the number is below the sampling percentage.
  */
 
-const FULLSTORY_RECORD_KEY = 'FULLSTORY_RECORD';
-
 // Percentage of overall users that we want to record
 const FULLSTORY_RECORD_PERCENT = 1 / 100;
 
+const FULLSTORY_RECORD_KEY = 'FULLSTORY_RECORD';
+
 enum RecordSession {
   TRUE = 'true',
   FALSE = 'false',
 }
 
-export function initFullStory() {
+function isRecordingEnabled() {
   if (!storageAvailable('localStorage')) {
-    return;
+    return false;
   }
-
   initStorage();
+  return localStorage.getItem(FULLSTORY_RECORD_KEY) === RecordSession.TRUE;
+}
 
-  // If we can't find the value in localStorage we won't enable recordings in FullStory
-  if (localStorage.getItem(FULLSTORY_RECORD_KEY) === RecordSession.TRUE) {
-    FullStory.init({ orgId: 'XEVN9' });
-  }
+async function getFullStory() {
+  return isRecordingEnabled() ? import('@fullstory/browser') : null;
+}
```

Comment:
FWIW- reading from localStorage is more expensive than reading from a local variable (localStorage is synced across tabs, etc.) and I don't know if `import` has any overhead to worry about, but it's tempting to cache the result of `getFullStory()`, especially since I assume this gets called several times on page load (to track the pageview, perf metrics, etc.)...

Probably over-optimization though, so feel free to punt. 😁

Reply:
will punt for now, I want to make the analytics tracking logic a bit more unified